core/src/test/java/org/elasticsearch/index/query/AbstractQueryTestCase.java

```diff
@@ -158,11 +157,6 @@
     private static int queryNameId = 0;
     private static SearchModule searchModule;
 
-
-    protected static QueryShardContext queryShardContext() {
-        return queryShardContext;
```

Comment:
no  I like creating a new instance, seems to reflect what we do everywhere at this point. But we should clean things up a little bit more, to make sure that the set search context is consistent at least.